sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/distillation/distillation_types.py

```diff
@@ -0,0 +1,308 @@
+# ---------------------------------------------------------
```

Comment:
dsitillation_types does not resonate greatly with me. Could we create separate files for each of the entity classes? Same for schema classes as well

Reply:
Gave each one their own file